src/attr/_funcs.py

```diff
@@ -63,6 +63,67 @@ def asdict(inst, recurse=True, filter=None, dict_factory=dict,
     return rv
 
 
+def astuple(inst, recurse=True, filter=None, tuple_factory=tuple,
+            retain_collection_types=False):
+    """
+    Return the ``attrs`` attribute values of *inst* as a tuple.
+
+    Optionally recurse into other ``attrs``-decorated classes.
+
+    :param inst: Instance of an ``attrs``-decorated class.
+    :param bool recurse: Recurse into classes that are also
+        ``attrs``-decorated.
+    :param callable filter: A callable whose return code determines whether an
+        attribute or element is included (``True``) or dropped (``False``).  Is
+        called with the :class:`attr.Attribute` as the first argument and the
+        value as the second argument.
+    :param callable tuple_factory: A callable to produce tuples from.  For
+        example, to produce ordered tuples instead of normal Python tuples.
```

Comment:
"to produce ordered tuples instead of Python tuples"? :D

Maybe change it to "to produce lists instead of tuples"